src/main/java/me/zeroeightsix/kami/module/modules/misc/DiscordRPC.kt

```diff
@@ -119,6 +119,10 @@ object DiscordRPC : Module() {
             LineInfo.DIMENSION -> {
                 InfoCalculator.dimension()
             }
+            LineInfo.TASK -> {
+                if (HighwayTools.isEnabled) "Making highways"
```

Comment:
I think we can make this more useful, maybe something like:

"Building highways for X seconds (Y blocks/hour)"

or something like that. @Avanatiker thoughts?

Reply:
Yes but i saw that it didn't show it in discord cus the line is kami blue is so short.